src/resource.js

```diff
@@ -207,11 +207,11 @@ angular.module('hypermedia')
        * @returns a promise that is resolved to the resource
        * @see Resource#$syncTime
        */
-      $load: {value: function () {
-        if (this.$syncTime) {
-          return $q.when(this);
-        } else {
+      $load: {value: function (ts) {
```

Comment:
I think you forgot to document the new function argument. (Also in `context.js`.)